src/modules/recipes/Categories/index.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { fetchCategories } from "../api";
+import "./styles.css";
+
+export const Categories = () => {
+  const [categories, setCategories] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    const getCategories = async () => {
+      try {
+        const data = await fetchCategories();
```

Comment:
Always add an extra validation rule to API requests to check if response was ok or not:
```
  if (!res.ok) {
    throw new Error("Failed to fetch categories");
  }
```